cou/cli.py

```diff
@@ -154,24 +146,14 @@ async def get_upgrade_plan(model_name: Optional[str], backup_database: bool) ->
     )
 
 
-async def run_upgrade(
-    model_name: Optional[str],
-    backup_database: bool,
-    prompt: bool,
-    quiet: bool,
-) -> None:
+async def run_upgrade(args: Namespace) -> None:
     """Run cloud upgrade.
 
-    :param model_name: Model name inputted by user.
-    :type model_name: Optional[str]
-    :param backup_database: Whether to create database backup before upgrade.
-    :type backup_database: bool
-    :param prompt: Whether to prompt to run upgrade interactively.
-    :type prompt: bool
-    :param quiet: Whether to run upgrade in quiet mode.
-    :type quiet: bool
+    :param args: CLI arguments
+    :type args: Namespace
     """
-    analysis_result, upgrade_plan = await analyze_and_plan(model_name, backup_database)
+    prompt = not args.auto_approve
```

Comment:
Can we do this in the `Namespace`? So here I can do simply `args.prompt`. If yes, I think that the `Namespace` could be renamed to something even different from `CLIargs` (from my comment below).